docs/conf.py

```diff
@@ -45,6 +45,7 @@
     'sphinx_markdown_tables',
     'symbolator_sphinx',
     'sphinxcontrib_verilog_diagrams',
+    'm2r'
```

Comment:
@mithro I'm using this extension to convert markdown into rst so that we can directly include README.md into the home page index.rst. However, there is a bug with the current version of m2r installed from pip/conda. I've applied the fix suggested [here](https://github.com/miyakogi/m2r/pull/55) to [my own fork](https://github.com/daniellimws/m2r). May I transfer it to SymbiFlow for sustainability purposes?

Also, this causes the readthedocs build to fail for now because of these lines https://github.com/SymbiFlow/python-symbiflow-v2x/pull/42/files#diff-85987f48f1258d9ee486e3191495582dR57-R59 causing there to be two markdown source parsers (when there should only be one).

Reply:
I think we need to figure out if we want to use `m2r` or `recommonmark` modules in all our documentation. This probably is a discussion for IRC.

Another option is to convert the `README.md` to a `README.rst` at the top level. Both PyPi and GitHub can reader rst files.